tests/Doctrine/Tests/DBAL/Functional/Schema/PostgreSqlSchemaManagerTest.php

```diff
@@ -348,6 +348,25 @@ public function testListQuotedTable() : void
         self::assertNull($comparator->diffTable($offlineTable, $onlineTable));
     }
 
+    public function testListTableDetailsWithNumberSchema() : void
+    {
+        $this->connection->exec('CREATE SCHEMA IF NOT EXISTS "001_test"');
```

Comment:
Would it be possible to rework the test to use the DBAL API (e.g. `(new Schema())->toSQL($platfrom)`) instead of platform-specific SQL and test it across all supported platforms?

We want to make sure that all the supported features are consistently implemented or at least tested across all supported platforms. IIRC, there was a similar issue reported for MySQL (something similar to https://github.com/doctrine/dbal/pull/2772).

Reply:
i can simplify this test case to
```php
public function testListTableDetailsWithNumberSchema() : void
{
    $this->connection->exec('CREATE SCHEMA IF NOT EXISTS "001_test"');
    $this->connection->exec('SET search_path = "001_test", public');

    try {
        $this->testListQuotedTable();
    } finally {
        $this->connection->exec('SET search_path TO DEFAULT');
    }
}
```

Reply:
Please do.

Reply:
done, also i changed target branch to 2.10